<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsciiOptionsWidget</class>
 <widget class="QWidget" name="AsciiOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lDateTimeFormat">
     <property name="text">
      <string>DateTime Format:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QCheckBox" name="chbSkipEmptyParts">
     <property name="text">
      <string>Skip empty Parts</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QCheckBox" name="chbConvertNaNToZero">
     <property name="text">
      <string>Convert NaN to 0</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="KComboBox" name="cbCommentCharacter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lColumnMode">
     <property name="text">
      <string>Column Datatypes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Comment Character:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lDecimalSeparator">
     <property name="text">
      <string>Decimal Separator:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QCheckBox" name="chbSimplifyWhitespaces">
     <property name="toolTip">
      <string>Removes the whitespaces from the start and the end, and replaces each sequence of internal whitespaces with a single space.</string>
     </property>
     <property name="text">
      <string>Simplify Whitespaces</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="5">
    <widget class="QLineEdit" name="kleVectorNames">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="KComboBox" name="cbDateTimeFormat">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cbDecimalSeparator"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lSeparatingCharacter">
     <property name="text">
      <string>Column Separator:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="5">
    <widget class="QLineEdit" name="kleColumnMode"/>
   </item>
   <item row="4" column="6">
    <widget class="QSpinBox" name="sbYearBase">
     <property name="toolTip">
      <string>Base year when only 2 digits are provided for the year</string>
     </property>
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>1900</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Year Base:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="lVectorNames">
     <property name="text">
      <string>Custom Names:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QCheckBox" name="chbRemoveQuotes">
     <property name="text">
      <string>Remove Quotes</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="KComboBox" name="cbSeparatingCharacter">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chbHeader">
     <property name="text">
      <string>Names at Line:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="sbHeaderLine">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="chbCreateTimestamp">
     <property name="text">
      <string>Create Timestamp </string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="chbCreateIndex">
     <property name="text">
      <string>Create Index</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QCheckBox" name="chbIntAsDouble">
     <property name="text">
      <string>Import Integer as Double</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
